sent2vec/vectorizer.py

```diff
@@ -40,8 +41,10 @@ def bert(self, sentences, pretrained_weights='distilbert-base-uncased'):
         self.vectors = vectors
 
     def word2vec(self, words, pretrained_vectors_path, ensemble_method='average'):
-        model = gensim.models.KeyedVectors.load_word2vec_format(pretrained_vectors_path)
-
+        if pretrained_vectors_path[-3:] == 'bin':
```

Comment:
Thanks for your work. To extract the extension of a file, you better use specific libraries for this purpose. This is not readable. 

```
import os
filename, file_extension = os.path.splitext('/path/to/somefile.ext')
```
 Look at this: https://stackoverflow.com/questions/541390/extracting-extension-from-filename-in-python

You should change the code like this:

if file_extention == 'bin'
  do this
elif file_extention == 'txt'
  do that
else
   raise IOError ('MESSAGE')

Reply:
Ok, I will make the change and open a new PR.

Thanks for your input